plugins/package-managers/spm/src/funTest/assets/projects/synthetic/spm-expected-output-lib.yml

```diff
@@ -38,7 +38,7 @@ project:
         dependencies:
         - id: "SPM:apple:swift-atomics:1.1.0"
         - id: "SPM:apple:swift-collections:1.0.4"
-    - id: "SPM:apple:swift-nio-ssl:2.24.0"
+    - id: "SPM:apple:swift-nio-ssl:2.25.0"
```

Comment:
Just out of curiosity: Have you been able to repro this on you machine? I saw the issue with version `2.25.0` already the day before yesterday, but locally it was still giving 2.24.0.

Reply:
I haven't tried (as I was on Windows) and just copied & pasted the updated result from the CI log.